src/components/AssetHeader/AssetDescription.tsx

```diff
@@ -14,14 +14,24 @@ type AssetDescriptionProps = {
   assetId: AssetId
 }
 
+const DESCRIPTION_DEFAULT_HEIGHT = 75
+
 export const AssetDescription = ({ assetId }: AssetDescriptionProps) => {
   const translate = useTranslate()
   const [showDescription, setShowDescription] = useState(false)
+  const descriptionEl = useRef<HTMLDivElement | null>(null)
   const handleToggle = () => setShowDescription(!showDescription)
   const asset = useAppSelector(state => selectAssetById(state, assetId))
   const { name, description, isTrustedDescription } = asset || {}
   const query = useGetAssetDescriptionQuery(assetId)
   const isLoaded = !query.isLoading
+
+  // If the height of the description is higher than the collapse element when it's closed
+  // we should display the Show More button
+  const shouldDisplayToggleButton =
+    descriptionEl?.current?.clientHeight &&
+    descriptionEl.current.clientHeight > DESCRIPTION_DEFAULT_HEIGHT
```

Comment:
```suggestion
  const shouldDisplayToggleButton =
    descriptionEl?.current?.clientHeight ?? 0 > DESCRIPTION_DEFAULT_HEIGHT
```

Reply:
I did use `(descriptionEl?.current?.clientHeight ?? 0) > DESCRIPTION_DEFAULT_HEIGHT` which is was we really wanted